spec/requests/shopify/update_variant_spec.rb

```diff
@@ -5,7 +5,7 @@
   include_context 'shopify_helpers'
 
   let(:spree_product) { create(:product, name: 'Product Name') }
-  let!(:spree_variant) { create(:variant, product: spree_product, sku: 'susan') }
+  let(:spree_variant) { create(:variant, product: spree_product, sku: 'susan_sku') }
```

Comment:
Not really but there is a song that say susan_sku or something that sounds like that so it makes me giggle everytime I go back in this specs. 

It's the little something ;)